lib/jellyfish/component/hls/storage.ex

```diff
@@ -186,8 +178,8 @@ defmodule Jellyfish.Component.HLS.Storage do
          partial_sn: partial_sn,
          room_id: room_id
        }) do
-    :ets.insert(room_id, {@manifest_key, manifest})
-    :ets.insert(room_id, {@last_partial_key, {segment_sn, partial_sn}})
+    :ets.insert({__MODULE__, room_id}, {@manifest_key, manifest})
+    :ets.insert({__MODULE__, room_id}, {@last_partial_key, {segment_sn, partial_sn}})
```

Comment:
As we are using `{__MODULE__, room_id}` in multiple places I think we should a helper function for creating a ets table name.